src/redux/slices/newEventSlice.js

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const newEventSlice = createSlice({
```

Comment:
I don't think we would need 2 separate slices for events, you could delete the one from pr #4, but would be helpful to rename this to eventSlice